sql/catalyst/src/main/java/org/apache/spark/sql/connector/util/V2ExpressionSQLBuilder.java

```diff
@@ -188,6 +195,14 @@ protected String visitNot(String v) {
     return "NOT (" + v + ")";
   }
 
+  protected String visitAbs(String v) {
```

Comment:
It looks an overkill if we add one visit method for each function. How about just one
```
String visitSQLFunction(String funcName, String[] inputs) ...
```